example/helm/values-scylla.yaml

```
@@ -13,23 +13,45 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+defaults:
+  logging:
+    level: debug
+  image:
+    tag: "nightly"
```

maybe we should use `latest` to avoid unstable release that affect users experience in `get started`
```suggestion
    tag: "latest"
```